apps/wing-console/console/server/package.json

```diff
@@ -31,10 +31,10 @@
     "@types/lodash.uniqby": "^4.7.7",
     "@types/ws": "^8.5.5",
     "@vitest/coverage-c8": "^0.31.4",
-    "@wingconsole/error-message": "file:../../packages/error-message",
-    "@wingconsole/eslint-plugin": "file:../../tools/eslint-plugin",
-    "@wingconsole/tsconfig": "file:../../tools/tsconfig",
-    "bump-pack": "file:../../../../tools/bump-pack",
+    "@wingconsole/error-message": "workspace:../../packages/error-message",
```

Comment:
All of the workspace dependencies can be installed as if they were third party dependencies, ex: `pnpm i @wingconsole/error-message`. If they belong to the workspace, they'll be symlinked and appear as `"@wingconsole/error-message": "workspace:^"` in the package.json.

So, there's no need to specify the relative path to the package: we can just use `workspace:^`. Thing is, do we prefer to be explicit? I prefer not worrying about the relative paths... less clutter.

Reply:
I'm down to try it out!